src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
`ColorSupplier` instance can be created just once - you don't need to do it every time you call `getRandomBall()`.